src/google/protobuf/compiler/cpp/parse_function_generator.cc

```diff
@@ -412,8 +416,7 @@ ::_pbi::TcParser::GetTable<$hottest_type_name$>(),  // to_prefetch
       if (line_entries) format("\n");
       format("65535, 65535\n");
     }
-    if (ordered_fields_.empty() &&
-        !descriptor_->options().message_set_wire_format()) {
```

Comment:
This might have caused issues internally. We still need aux table for message set. Why did you drop this?